### PR TITLE
[SPARK-46673][PYTHON][DOCS] Refine docstring `aes_encrypt/aes_decrypt/try_aes_decrypt`

### PR DESCRIPTION
**Pull request by panbingkun**
### What changes were proposed in this pull request?
The pr aims to refine docstring of `aes_encrypt/aes_decrypt/try_aes_decrypt`.

### Why are the changes needed?
To improve PySpark documentation.

### Does this PR introduce _any_ user-facing change?
No.

### How was this patch tested?
- Pass GA.
- Manually test.

### Was this patch authored or co-authored using generative AI tooling?
No.

### CODE REVIEW

File: python/pyspark/sql/functions/builtin.py [hunk 1]
```diff
@@ -18698,6 +18698,8 @@ def nvl2(col1: "ColumnOrName", col2: "ColumnOrName", col3: "ColumnOrName") -> Co
     return _invoke_function_over_columns("nvl2", col1, col2, col3)
 
 
+# TODO(SPARK-46738) Re-enable testing that includes the 'Cast' operation after
```
Comment (panbingkun) [inline python/pyspark/sql/functions/builtin.py:18701]: Considering that cast is displayed differently in `Regular Spark` and `Spark Connect`, let's ignore the test with `cast` for now.  I have created a new jira (https://issues.apache.org/jira/browse/SPARK-46738) and I am currently trying to fix it. 
After fixing, I will open the relevant UT.

**Review by MaxGekk — COMMENTED** [review]
@sweisdb Could you review this PR, please, since you worked on the functionality.

**Review by MaxGekk — APPROVED** [review]

**Comment by MaxGekk** [comment]
+1, LGTM. Merging to master.
Thank you, @panbingkun.

**Comment by panbingkun** [comment]
I have submitted a fix pr for `SPARK-46738`: https://github.com/apache/spark/pull/44829

**Review by sweisdb — COMMENTED** [review]
Belated: This looked fine to me too.

**Pull request closed without merging**